src/backend/environment_aws.yml

```diff
@@ -0,0 +1,125 @@
+  - gstreamer=1.14.1=h5eee18b_1
+  - hdf4=4.2.13=h3ca952b_2
+  - hdf5=1.10.6=h3ffc7dd_1
+  - icu=73.1=h6a678d5_0
+  - jpeg=9e=h5eee18b_1
+  - jsoncpp=1.9.4=hdb19cb5_3
+  - krb5=1.20.1=h143b758_1
+  - lame=3.100=h7b6447c_0
+  - ld_impl_linux-64=2.38=h1181459_1
+  - lerc=3.0=h295c915_0
+  - libclang=14.0.6=default_hc6dbbc7_1
+  - libclang13=14.0.6=default_he11475f_1
+  - libcups=2.4.2=h2d74bed_1
+  - libcurl=8.2.1=h251f7ec_0
+  - libdeflate=1.17=h5eee18b_0
+  - libedit=3.1.20221030=h5eee18b_0
+  - libev=4.33=h7f8727e_1
+  - libevent=2.1.12=hdbd6064_1
+  - libffi=3.4.4=h6a678d5_0
+  - libgcc-ng=11.2.0=h1234567_1
+  - libgfortran-ng=11.2.0=h00389a5_1
+  - libgfortran5=11.2.0=h1234567_1
+  - libglu=9.0.0=hf484d3e_1
+  - libgomp=11.2.0=h1234567_1
+  - libidn2=2.3.4=h5eee18b_0
+  - libllvm14=14.0.6=hdb19cb5_3
+  - libnetcdf=4.8.1=h8322cc2_2
+  - libnghttp2=1.52.0=h2d74bed_1
+  - libogg=1.3.5=h27cfd23_1
+  - libopenblas=0.3.21=h043d6bf_0
+  - libopus=1.3.1=h7b6447c_0
+  - libpng=1.6.39=h5eee18b_0
+  - libpq=12.15=hdbd6064_1
+  - libssh2=1.10.0=hdbd6064_2
+  - libstdcxx-ng=11.2.0=h1234567_1
+  - libtasn1=4.19.0=h5eee18b_0
+  - libtheora=1.1.1=h7f8727e_3
+  - libtiff=4.5.1=h6a678d5_0
+  - libunistring=0.9.10=h27cfd23_0
+  - libuuid=1.41.5=h5eee18b_0
+  - libvorbis=1.3.7=h7b6447c_0
+  - libvpx=1.7.0=h439df22_0
+  - libwebp-base=1.3.2=h5eee18b_0
+  - libxcb=1.15=h7f8727e_0
+  - libxkbcommon=1.0.1=h5eee18b_1
+  - libxml2=2.10.4=hf1b16e4_1
+  - libzip=1.8.0=h6ac8c49_1
+  - loguru=0.5.3=py39h06a4308_4
+  - lz4-c=1.9.4=h6a678d5_0
+  - mysql=5.7.24=h721c034_2
+  - ncurses=6.4=h6a678d5_0
+  - nettle=3.7.3=hbbd107a_1
+  - nomkl=3.0=0
+  - nspr=4.35=h6a678d5_0
+  - nss=3.89.1=h6a678d5_0
+  - numpy=1.20.3=py39h3dbb7de_0
+  - numpy-base=1.20.3=py39h39b7dee_0
+  - openblas=0.3.21=h06a4308_0
+  - openblas-devel=0.3.21=h06a4308_0
+  - openh264=2.1.1=h4ff587b_0
+  - opensim=4.4.1=py39np120
+  - openssl=3.0.10=h7f8727e_2
+  - pcre=8.45=h295c915_0
+  - pip=23.2.1=py39h06a4308_0
+  - proj=6.2.1=h05a3930_0
+  - python=3.9.18=h955ad1f_0
+  - qt=5.15.9=h06a4308_0
+  - qt-main=5.15.2=h53bd1ea_10
+  - qt-webengine=5.15.9=h9ab4d14_7
+  - readline=8.2=h5eee18b_0
+  - setuptools=68.0.0=py39h06a4308_0
+  - sqlite=3.41.2=h5eee18b_0
+  - tbb=2021.8.0=hdb19cb5_0
+  - tk=8.6.12=h1ccaba5_0
+  - tzdata=2023c=h04d1e81_0
+  - utfcpp=3.2.1=h06a4308_0
+  - vtk=9.0.3=py39ha4caa0b_203
+  - wheel=0.38.4=py39h06a4308_0
+  - x264=1!157.20191217=h7b6447c_0
+  - xz=5.4.2=h5eee18b_0
+  - zlib=1.2.13=h5eee18b_0
+  - zstd=1.5.5=hc292b87_0
+  - pip:
+      - asgiref==3.7.2
+      - awslambdaric==2.0.7
+      - dataclasses-json==0.6.0
+      - deprecated==1.2.14
+      - marshmallow==3.20.1
+      - mypy-extensions==1.0.0
+      - packaging==23.1
+      - pygltflib==1.16.0
+      - pytz==2023.3.post1
+      - sqlparse==0.4.4
+      - typing-extensions==4.8.0
+      - typing-inspect==0.9.0
+      - urllib3==2.0.2
+      - wrapt==1.15.0
+prefix: /home/ayman/miniconda3/envs/osv_bend
```

Comment:
This prefix is pointing to a path in a specific machine.

Reply:
Sure, thanks for the find and please remove as we discussed, particularly the aws.yml file is not used by aws altogether